regulations/html.py

```diff
@@ -56,6 +52,19 @@ def __init__(self, language, buildDir, pdfName, isTranslation=False, verbose=Fal
     if not self.isTranslation:
       self.pages()
 
+  def __del__(self):
+    self.regulations_fp.close()
+
+  def find_matching_reg(self, reg_id):
+    # Go back to the beginning of regulations file
+    self.regulations_fp.seek(0)
```

Comment:
Sweet, thanks! I mostly want to do it because we can use python's `with` operator this way, and get rid of the scary looking `__del__` method =)